packages/model/test/BaseResource.test.ts

```diff
@@ -0,0 +1,68 @@
+import { describe, it } from 'mocha'
+import RdfResource from '@tpluscode/rdfine/RdfResource'
+import { blankNode } from '@cube-creator/testing/clownface'
+import { fromPointer } from '@rdfine/schema/lib/Thing'
+import { expect } from 'chai'
+import { schema } from '@tpluscode/rdf-ns-builders/strict'
+import { literal } from '@rdf-esm/data-model'
+import { BaseResourceMixin } from '../BaseResource'
+
+describe('@cube-creator/model/BaseResource', () => {
+  describe('addError', () => {
+    it('adds error when new', () => {
```

Comment:
Just for curiosity, why are errors added as RDF?

Reply:
We only have a triple store to persist. How would you add them?

Reply:
That was more of a question to understand the app. 

I also was trying to understand if we need to store things like errors. (Not particularly for this pull request)

Reply:
I have the feeling that persisting errors can become (in the future) something difficult to manage. The drawback I see is that in our code, we will have to check:

- What are the current errors? 
- Are the errors still valid?
- Can we delete the errors? 

I don't have a clear idea yet about the model, so I'm not sure if the following applies:

 I imagine a draft area where one builds up entities (cube, dimension mapping, etc.) to be created in RDF later.

The app validates everything in the draft area with, for example, SHACL; messages are calculated and displayed at runtime (never persisted). 

If the user decides to create a 'valid' element, it is promoted from draft to 'real' in RDF.